vectorize-sharepoint-aisearch-schedule/manifest.json

```diff
@@ -0,0 +1,81 @@
+{
+    "title": "Vectorize files on a schedule from OneDrive for Business to AI Search",
+    "description": "This workflow template runs using the schedule-based Recurrence trigger and vectorizes files from a  Microsoft OneDrive for Business folder and writes them to the AI Search vector store.",
+    "prerequisites": "Before you use this template, you need to configure an Azure OpenAI instance with an embedding model. You also need to configure an AI Search instance as your vector store along with an index and the appropriate schema. The index schema is based on the [Azure AI Search indexer default schema](https://go.microsoft.com/fwlink/?linkid=2305401).",
+    "tags": [
+        "RAG-Document-Ingestion"
+    ],
+    "skus": [
+        "consumption"
+    ],
+    "kinds": [
+    ],
+    "detailsDescription": "This workflow template processes and indexes a document using Azure OpenAI and Azure AI Search. The workflow uses the scheduled-based Recurrence trigger, which you can set up to run using a specific interval and frequency. The workflow gets the document, parses the content, and chunks the text based on token size. The workflow then processes each chunk to generate embeddings through an OpenAI deployment. The text chunks and their embeddings are mapped to a schema suitable for Azure AI Search. Finally, the documents are indexed in Azure AI Search for efficient retrieval and analysis. This template is built on AI building blocks in Azure Logic Apps, and includes the Azure OpenAI connector, Azure AI Search connector, and more. The template also uses built-in actions for document parsing and chunking. For more information, see [Connect to Azure AI services from workflows in Azure Logic Apps](https://go.microsoft.com/fwlink/?linkid=2304933).",
+    "details": {
+        "By": "Microsoft",
+        "Type": "Workflow",
+        "Trigger": "Recurrence",
+        "Category": "AI,RAG"
+    },
+    "artifacts": [
+        {
+            "type": "workflow",
+            "file": "workflow.json"
+        }
+    ],
+    "images": {
+        "light": "workflow-light",
+        "dark": "workflow-dark"
+    },
+    "parameters": [
+        {
+            "name": "FolderName_#workflowname#",
+            "displayName": "OneDrive for Business folder name",
```

Comment:
```suggestion
            "displayName": "SharePoint Online folder name",
```

Reply:
removed this , was not needed in this template